partner_industry_parent/models/res_partner_industry.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartnerIndustry(models.Model):
+
+    _inherit = 'res.partner.industry'
+    _parent_name = "parent_id"
+    _parent_store = True
+    _rec_name = 'complete_name'
+    _order = 'complete_name'
+
+    complete_name = fields.Char(
```

Comment:
You might consider to use the display_name instead of creating a separate field. If you rather use complete_name, should it not be visible in the form you create later? Because there might be multiple levels...

Reply:
Complete_name is different than display_name. It contain the complete path for the current object to the first ancestor. Display name can contain the object name with its code for example.

Complete_name is used almost in every use case odoo standard set a parent store to true (hr.department, product.category..). So IMO, it's better to keep the same logic.

Reply:
@sbejaoui OK, but that leaves my remark on showing somewhere the complete name, at least in the form.

Reply:
ok you are right, I missed that point. I pushed a new commit to add the complete name in the form view.
Thank you